api/v1alpha3/types.go

```diff
@@ -437,3 +437,9 @@ type SecurityProfile struct {
 	// set. Default is disabled.
 	EncryptionAtHost *bool `json:"encryptionAtHost,omitempty"`
 }
+
+// Record specifies a DNS record.
+type Record struct {
```

Comment:
Is this a specific type of DNS record, a cname, A record, etc?

Reply:
it's A or AAAA (mapping hostname to IP v4 or v6). Tried to clarify in comments and renamed name to hostname.

Reply:
I believe these types of records are specifically known as address records. Would you consider `AddressRecord`?

https://en.wikipedia.org/wiki/List_of_DNS_record_types

Reply:
good idea, done